projects/smart-table/src/lib/smart-table.types.ts

```diff
@@ -10,6 +9,7 @@ export interface IModuleConfig {
 }
 
 export interface ILabels {
+  sorting?: { sortAbove: string, sortUnderneath: string };
```

Comment:
Is eerder "ordering" dan "sorting". De naamgeving is verwarrend omdat sortering ook een feature is.